timed/tracking/filters.py

```diff
@@ -71,17 +71,33 @@ def filter(self, qs, value):
         ).distinct()
 
 
+class RecentActivityFilter(Filter):
+    """Filter to get recent activities."""
+
+    @boolean_filter
```

Comment:
would it be better to use an integer filter so the amount of recent items can actually be given?

Would be more flexible also for features where a recent item settings can be set by user.